egs/ami/s5b/local/chain/tuning/run_cnn_tdnn_lstm_1c.sh

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
+
+# cnn_tdnn_lstm_1c is based on cnn_tdnn_lstm_1b, but using smaller dropout-schedule
+# and larger decay-time option(40).
+
+# After testing different combinations of dropout-schedule('0,0@0.20,0.15@0.50,0',
+# '0,0@0.20,0.3@0.50,0' or without), proportional-shrink(with or without) and 
+# decay-time option(20, 40 or without), we found this setup is best.
+
+# SDM
+# Results with flags : --mic sdm1 --use-ihm-ali true --train-set train_cleaned  --gmm tri3_cleaned \
+# ./local/chain/compare_wer_general.sh sdm1 tdnn_lstm1j_sp_bi_ihmali_ld5 cnn_tdnn_lstm1b_sp_bi_ihmali_ld5 cnn_tdnn_lstm1b_sp_bi_ihmali_ld5_online
```

Comment:
Thanks. I will change it.
BTW, local/chain/compare_wer_general.sh cannot deal with online directory. Because, for the online directory, the decode directory is named "decode_online_dev" and it doesn't have log directory.